.travis.sh

```diff
@@ -111,6 +119,9 @@ travis_before_deploy()
     if [ -z "$ANDROID_KEYSTORE_PASS" ]; then
         return
     fi;
+    if [ "$TARGET_OS" = "Linux" ]; then
+        cp ../../build_cmake/build/Play*.AppImage .
```

Comment:
@jpd002 it'd probably easier for you to test fix instead of, since PRs dont deploy, but looking at the logs, i think path should be `../../build/Play*.AppImage`?

Reply:
I've changed it, working good now.

Reply:
that's great :)
I tested the uploaded image and it works great on Ubuntu 16.04, will test on arch later and will report if anything worth mentioning.